app/models/health_question.rb

```diff
@@ -19,4 +19,24 @@
 class HealthQuestion < ApplicationRecord
   attr_accessor :response, :notes
   belongs_to :vaccine
+
```

Comment:
We should create a separate class that isn't backed by AR, possibly `HealthAnswer`